src/ceph.in

```diff
@@ -732,10 +731,10 @@ def main():
         print('Cluster connection aborted', file=sys.stderr)
         return 1
     except rados.PermissionDeniedError as e:
-        print('Error connecting to cluster: {0}'.format(e.__class__.__name__), file=sys.stderr)
+        print(str(e), file=sys.stderr)
```

Comment:
you removed the "Error connecting to cluster: " from the error message. is this intentional?